pages/cloudflare/howtos/NextAuth.mdx

```diff
@@ -28,3 +33,18 @@ Kudos to Arnav Gupta ([`@arnavgupta00`](https://github.com/arnavgupta00)) for co
 You can find an example of this on his [example repository](https://github.com/arnavgupta00/deployment-cf-workers-prisma-nextauth).
 
 Related issues: [`workers-sdk#206`](https://github.com/opennextjs/opennextjs-cloudflare/issues/206) and [`workerd#3277`](https://github.com/cloudflare/workerd/issues/3277).
+
+### Solving issues in local dev
+
+When trying to access bindings (for instance to use [D1](/d1/) as database) depending on your implementation, you might run into:
+
+`ERROR: getCloudflareContext has been called without having called initOpenNextCloudflareForDev from the Next.js config file.`
+
+You can resolve this issue, by following the the following pattern in your `auth.ts` configuration:
```

Comment:
This sentence is a bit off, besides the "the the", I think that the "by following the following pattern" sentence really doesn't flow nicely, I'd change the wording to something like this:

```suggestion
You can resolve this issue, by calling `getCloudflareContext` in your `NextAuth` callback, for example like so:
```